### PR TITLE
add basic skeleton for user created decks -- just decks

### PR DESCRIPTION
**Pull request by lksevans12**
@jkrth617 created a deck new view page to create a new deck, and add routes to deck

### CODE REVIEW

File: app/controllers/deck.rb [hunk 1]
```diff
@@ -6,4 +6,19 @@
...
+end
+
+get 'decks/new' do
+  @user = User.find(session[:user_id])
+  erb :'deck/new'
+end
+
+post '/decks' do
+  @deck = Deck.new(params[:deck])
+  @deck.id = session[:deck][:id]
```
Comment (jkrth617) [inline app/controllers/deck.rb]: db deals with this

File: app/controllers/deck.rb [hunk 2]
```diff
@@ -6,4 +6,19 @@
...
+  if @deck.save
+    redirect "/decks/#{@deck.id}/cards/new"
+  else
+    "not saved"
```
Comment (jkrth617) [inline app/controllers/deck.rb]: add a redirect eventually back to deck/new with errors so the user can try again

File: app/views/deck/new.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<h2> Great, so you want to create your own deck! </h2>
+<h2> First, please enter the name of your new deck. </h2>
+<form action = "/users/<%= @user.id %>/decks" method = "post">
```
Comment (jkrth617) [inline app/views/deck/new.erb]: change to post /decks

**Comment by jeder28** [comment]
yeay!! :dart:  :dancer:

**Pull request merged**